Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -892,8 +878,90 @@ theorem SeminormedGroup.uniformCauchySeqOn_iff_tendstoUniformlyOn_one {f : ι 
     uniformCauchySeqOn_iff_uniformCauchySeqOnFilter,
     SeminormedGroup.uniformCauchySeqOnFilter_iff_tendstoUniformlyOnFilter_one]
 
+variable [T0Space E]
```

Comment:
What's the motivation for swapping this for `NormedGroup`? The two spellings are equivalent, right?

Reply:
(phrased another way: this refactor looks out of scope for this PR, and if you're serious about wanting to avoid NormedGroup entirely in favor of the less-bundled working, probably deserves a Zulip thread)

Reply:
Yes, true. They are equivalent. Why do we even have lemmas stated with `SeminormedGroup` + `T0Space`?